src/ui/components/common/sidebar.rs

```diff
@@ -1,135 +1,152 @@
-use crate::{
-  ui::components::common::{
-    icon::{Icon, IconSize, IconType},
-    unpack::Unpack,
-  },
-  utils::types::QuerySignal,
-};
-use lemmy_client::lemmy_api_common::lemmy_db_schema::source::person::Person;
+use crate::ui::components::common::icon::{Icon, IconSize, IconType};
 use leptos::*;
 use leptos_fluent::move_tr;
 use pretty_num::PrettyNumber;
+use sidebar_data::SidebarData;
 use team_member_card::TeamMemberCard;
 use user_stat_row::UserStatRow;
 
-mod sidebar_data;
-pub use sidebar_data::SidebarData;
+pub mod sidebar_data;
 mod team_member_card;
 mod user_stat_row;
 
 #[component]
-pub fn Sidebar(data: QuerySignal<SidebarData>, team: QuerySignal<Vec<Person>>) -> impl IntoView {
+pub fn Sidebar<'a>(data: &'a SidebarData) -> impl IntoView {
```

Comment:
Always try to pass the simple objects into components. The Top-level "screens" or "pages" should be doing all the work of extracting API responses, and components should be taking in simple data objects.